docs/ccm/v0.4.x/ccm_configuration.md

```diff
@@ -0,0 +1,64 @@
+# Nutanix CCM Configuration
+
+Nutanix CCM can be configured via a `JSON` formated file stored in a configmap called `nutanix-config`. This configmap is located in the same namespace as the Nutanix CCM deployment. See the `manifests/cloud-provider-nutanix-deployment.yaml` file for details on the Nutanix CCM deployment. 
+
+Example `nutanix-config` configmap:
+```YAML
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: nutanix-config
+  namespace: kube-system
+data:
+  nutanix_config.json: |-
+    {
+      "prismCentral": {
+        "address": "${NUTANIX_ENDPOINT}",
+        "port": ${NUTANIX_PORT},
+        "insecure": ${NUTANIX_INSECURE},
+        "credentialRef": {
+          "kind": "secret",
+          "name": "nutanix-creds"
+        },
+        "additionalTrustBundle": {
+          "kind": "ConfigMap",
+          "name": "user-ca-bundle"
+        }
+      },
+      "enableCustomLabeling": false,
+      "topologyDiscovery": {
+        "type": "Categories",
+        "topologyCategories": {
+          "regionCategory": "${NUTANIX_REGION_CATEGORY}",
+          "zoneCategory": "${NUTANIX_ZONE_CATEGORY}"
+        }
+      }
+    }
+
+```
+
+The table below provides an overview of the supported configuration parameters.
+
+### Configuration parameters
+
```

Comment:
We are missing https://github.com/nutanix-cloud-native/cloud-provider-nutanix/pull/187/files#diff-ad73b409347f06a6b8368277cd6c8419c9daee59a55b332bfd517ff3cf311a6a